main.tex

```diff
@@ -114,6 +114,12 @@ \subsection{Übersicht}
 In der Zwischenzeit versuchte Facebook, nach initellen Startschwierigkeiten, die Initiative durch Rebranding zu retten, was aber nicht reichte um die kritische Öffentlichkeit zu beeinflußen, deren Meinung bis zu den Entscheidungträgern der zuständigen Behörde vordrigen konnte.
 Welche sich dann mit einem indirekten Verbot auf der Seite der Protestierenden positionierte.
 
-% \printbibliography
+% Fazit
+\section{Fazit}
+
+Aus unseren zusammengetragenen Informationen lässt sich erschließen, dass sich das Projekt Internet.org/Free Basic in Indien in kurzer Zeit schnell negativ entwickelt hat.
+Zwischen dem formellen Start im Februar 2015 und dem Verbot im Februar 2016 ist gerade mal ein Jahr vergangen.
+Dadurch wird deutlich, dass innerhalb des Jahres viele Probleme an dem Projekt auftraten.
```

Comment:
Aus unseren zusammengetragenen Informationen lässt sich erschließen, dass das Projekt Internet.org/Free Basics in Indien in kurzer Zeit (Start Februar 2015; Verbot Februar 2016) unter anderem durch negative öffentliche Meinungsäußerung gescheitert ist. Der von Facebook angezielte Markt war durch ein ähnliches und bereits unter Feuer geratenes Projekt nur Monate vorher verstimmt worden, fehlerhaftes Management, insbesondere die Stimme an Presse und Bevölkerung nahm nur selten an der Diskussion um tatsächlich besprochene Kitikpunke teil, wobei selbst dann nur das eigene Projekt verteidigt wurde. Als im September endlich nachgelassen wurde, war es bereits zu spät; die Bevölkerung stimmte der Opposition zu, was final in der Entscheidung des TRAI resultierte.